ner4opt/ner4opt.py

```diff
@@ -146,6 +148,9 @@ def get_entities(self, text: str) -> List[dict]:
         if not text:
             return predicted_entities
 
+        # preprocess
+        text = preprocess(text)
```

Comment:
learning q -- how does this work with or relate to our L2 augmentation? That happens later on? And what happens to L2 when it cannot find an objective min/max?  OR.. it does not care about that, simply grabs last 2 regardless?

Reply:
Yes, l2_augment is not affected by this. We augment regardless of whether we find an objective name or not. We assume that it is a fundamental property of pretty much all our problems, at least for our case (training data and dev data)